tap_asana/streams/portfolios.py

```diff
@@ -28,15 +27,13 @@ class Portfolios(Stream):
 
 
   def get_objects(self):
-    bookmark = self.get_bookmark()
-    session_bookmark = bookmark
     opt_fields = ",".join(self.fields)
-    for workspace in Context.asana.client.workspaces.find_all():
+    for workspace in Context.asana.client.workspaces.find_all(page_size=self.results_per_page):
```

Comment:
@kspeer825 We have added support of page size as part of the Pagination Integration Test. Previously during the API call the `page_size` was not passed by default and now if the user does not enter the page size it will be passed in all the API calls with value `50`, is this the right behaviour?

CC: @dbshah1212 @karanpanchal-crest